src/vaadin-combo-box-mixin.html

```diff
@@ -623,10 +624,29 @@
       }
     }
 
+    _itemLabelPathChanged(itemLabelPath, oldItemLabelPath) {
```

Comment:
There should already be functionality somewhere that refreshes the item labels since they're updated for example when the items change. No need to write new logic for it but instead make sure that the existing logic gets invoked also on `itemLabelPath` change

Reply:
Done